.gitignore

```diff
@@ -70,3 +70,9 @@ yarn-debug.log*
 # Sitemap is generated periodically
 public/sitemap.xml*
 public/*.sidecar_report.csv
+
+# Ignore Python virtual environment
+lib/opendataharvest/venv/
```

Comment:
It's best practice _not_ to commit venv to version control. Instead, I've created a new rake task to run the python setup script which will create a venv appropriate for the system where it's being installed.